storage/storagedriver/select-storage-driver.md

```diff
@@ -73,25 +78,34 @@ In addition, Docker does not recommend any configuration that requires you to
 disable security features of your operating system, such as the need to disable
 `selinux` if you use the `overlay` or `overlay2` driver on CentOS.
 
-### Docker EE and CS-Engine
+### Docker Engine Enterprise and Docker EE
```

Comment:
shd be "Docker Engine - Enterprise" and "Docker Enterprise"

Reply:
fixed throughout.